src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1755,23 +1761,39 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
     }
 
 
-    // === Node Edit Mode ===
+    // === Node Editing ===
 
-    frp::extend! { network
-        edit_mode    <- bool(&inputs.edit_mode_off,&inputs.edit_mode_on);
-        node_to_edit <- touch.nodes.selected.gate(&edit_mode);
-        eval node_to_edit ([model](id) {
-            if let Some(node) = model.nodes.get_cloned_ref(id) {
+    frp::extend! { TRACE_ALL network
+        is_edited                <- outputs.edited_node.map(|n| n.is_some());
+        edit_mode                <- bool(&inputs.edit_mode_off,&inputs.edit_mode_on);
+        node_edited_in_edit_mode <- touch.nodes.selected.gate(&edit_mode);
+        start_editing            <- any(&node_edited_in_edit_mode,&inputs.edit_node);
+         // FIXME: add other cases like node select.
```

Comment:
i dont understand this fixme - please describe it better OR implement it OR create issue for it

Reply:
It's **your** FIXME. If you don't remember this, I will remove it.